netlify.toml

```diff
@@ -977,3 +977,153 @@ status = 301
+  to = "/account/my-organization/integrations/notification-integrations/email-connection"
+  status = 301
+
+[[redirects]]
+  from = "/account/my-organization/api-integrations/adding-an-app-store-connect-api-key"
+  to = "/account/my-organization/integrations/credentials/adding-an-app-store-connect-api-key"
+  status = 301
+
+[[redirects]]
+  from = "/account/my-organization/api-integrations/adding-google-play-service-account"
+  to = "/account/my-organization/integrations/credentials/adding-google-play-service-account"
+  status = 301
+
+[[redirects]]
+  from = "/account/my-organization/api-integrations/adding-huawei-api-key"
+  to = "/account/my-organization/integrations/credentials/adding-huawei-api-key"
+  status = 301
+
+[[redirects]]
+  from = "/account/my-organization/api-integrations/adding-microsoft-intune-api-key"
+  to = "/account/my-organization/integrations/credentials/adding-microsoft-intune-api-key"
+  status = 301
+
+[[redirects]]
+  from = "/account/my-organization/role-management"
+  to = "/account/my-organization/profile-and-team/role-management"
+  status = 301
+
+[[redirects]]
+  from = "/account/my-organization/sso-providers-configuration/auth-openid"
+  to = "/account/my-organization/integrations/authentications/sso-providers-configuration/auth-openid"
+  status = 301
+
+[[redirects]]
+  from = "/account/my-organization/sso-providers-configuration/auth-saml"
+  to = "/account/my-organization/integrations/authentications/sso-providers-configuration/auth-saml"
+  status = 301
+
+[[redirects]]
+  from = "/account/my-organization/sso-providers-configuration/azure-saml"
+  to = "/account/my-organization/integrations/authentications/sso-providers-configuration/azure-saml"
+  status = 301
+
+[[redirects]]
+  from = "/account/my-organization/sso-providers-configuration/okta-openid"
+  to = "/account/my-organization/integrations/authentications/sso-providers-configuration/okta-openid"
+  status = 301
+
+[[redirects]]
+  from = "/account/my-organization/sso-providers-configuration/okta-saml"
+  to = "/account/my-organization/integrations/authentications/sso-providers-configuration/okta-saml"
+  status = 301
+
+[[redirects]]
+  from = "/account/my-organization/sso-providers-configuration/onelogin-saml"
+  to = "/account/my-organization/integrations/authentications/sso-providers-configuration/onelogin-saml"
+  status = 301
+
+[[redirects]]
+  from = "/account/my-organization/api-integrations/adding-an-app-store-connect-api-key#login-to-app-store-connect"
+  to = "/account/my-organization/integrations/credentials/adding-an-app-store-connect-api-key#login-to-app-store-connect"
+  status = 301
+
+[[redirects]]
+  from = "/account/my-organization/notifications-and-communication"
+  to = "/account/my-organization/integrations/notification-integrations"
+  status = 301
+
+[[redirects]]
+  from = "/account/my-organization/role-management#publish-environment-variables"
```

Comment:
For this and similar redirect header updates, if the base URL is updated and the header has not changed, is it necessary to redirect the header?